XenAdmin/Dialogs/VMProtectionRecovery/PolicyHistory.cs

```diff
@@ -169,7 +169,13 @@ private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
             if (_policy != null)
             {
                 if (comboBox1.SelectedIndex == 0)
-                    RefreshGrid(_policy.PolicyAlerts);
+		{
+                    dataGridView1.Rows.Clear();

```

Comment:
This sequence (ln 173-177) is repeted in each if block with the only difference on the last argument of the getAlertsAction() function. 
It could be optimised to just use a different value for that parameter, depending on comboBox1.SelectedIndex.